samples/custom_dispatch/cpu/embedded/example_hal.mlir

```diff
@@ -41,18 +41,27 @@
 // These can come from compiler flags and multiple targets can be supported
 // It's possible, for example, to support targeting multiple devices in the same
 // compiled binary (CPU + Vulkan, etc).
-#cpu_target = #hal.device.target<"llvm-cpu", [
+#cpu_target = #hal.device.target<"local", {
+  executable_create_2
+}, [
```

Comment:
mlir syntax highlighting colors the `_2` differently here 🤔 
is there a syntax error there?

Reply:
nah, probably just a bad highlighter - maybe the highlighter is assuming any trailing number is a uniqueing value, but here it's a unit attr with a string key that has arbitrary syntax
(by the end of this PR change this will be deleted anyway)